docs/instructor-training-workflow.txt

```diff
@@ -0,0 +1,174 @@
+    - Adds the <Training Application> to <Training Application List>.
+
+## Reviewing training queue.
+
+* [admin reviews training queue]: to see who's waiting and how long.
+  - Admin goes to {training queue display}.
+  - Admin selects filtering and sorting criteria (if any).
+  - Admin pages through display showing both <Training Application>s and <Group Training Application>s.
+
+## Setting up training course.
+
+* [admin creates training event]: and assigns people to it.
+  - Admin creates new <Event>.
+  - Admin goes to {training queue display}.
+  - Admin selects <Event>.
+  - Admin selects some mix of <Training Application> and <Group Training Application>.
+  - AMY creates <Task>s to add all the selected <Person>s to the <Event>.
+  - AMY connects the <Training Application> and <Group Training Application> to the <Event>.
+  - AMY moves <Training Application>s and <Group Training Application>s to <Assigned Application List>.
+  - AMY provides single-click "send to all" email link to admin to inform trainees.
+  - AMY creates <Training Progress> record for each <Person>.
+
+## Administering training course applications.
+
+* [admin removes people from training event][a]: and discards applications
+  - Admin goes to {assigned training application display}.
+  - Admin selects <Training Application>s and <Group Training Application>s.
+  - Admin removes them from training event.
+  - AMY deletes associated <Task>s.
+
+* [admin removes people from training event][b]: but keeps applications
+  - => [admin removes people from training event][a]
+  - AMY moves <Training Application>s and <Group Training Application>s back to <Training Application List>.
+
+## Reviewing training progress.
+
+* [admin views training progress]: both past and present.
+  - Admin views {training progress page}.
+  - Admin selects filtering and sorting criteria (if any).
+
+## Tracking submission of written work.
+
+* [trainee reports submission of written work]: such as pull request or comment on lesson.
+  - Trainee views their own {training progress}.
+  - Trainee enters URL of written work.
+  - AMY updates <Training Task> associated with <Person> and <Event>.
+  - AMY adds <Training Progress> to <Training Progress Review List>.
+
+* [trainer reports submission of written work]: on behalf of trainee.
+  - Trainer looks up <Training Progress> associated with trainee.
+  - Trainer enters URL of written work in <Training Task>.
+
+* [trainer reports submission of written work]: on behalf of trainee.
+  - Trainer looks up <Training Progress> associated with trainee.
+  - Trainer enters URL of written work in <Training Task>.
+  - AMY adds <Training Progress> to <Training Progress Review List>.
+
+* [trainer reports completion of written work]: on behalf of trainee.
+  - Trainer looks up <Training Progress> associated with trainee.
+  - Trainer enters URL of written work in <Training Task>.
+  - Trainer marks <Training Task> as complete.
```

Comment:
It may be me misunderstanding. Why does the trainer have to report submission of trainee's written work on the trainee's behalf? This should be submitted by the trainee themselves, right?